ui/src/forms/components.ts

```diff
@@ -1,115 +1,96 @@
-import { html, SingleEditorComponent } from '@hydrofoil/shaperone-wc'
+import { html, SingleEditorComponent, Lazy } from '@hydrofoil/shaperone-wc'
 import * as ns from '@cube-creator/core/namespace'
 import { dash, xsd } from '@tpluscode/rdf-ns-builders'
 import $rdf from '@rdfjs/data-model'
 import { createCustomElement } from '@/forms/bulma'
 
-export const textField: SingleEditorComponent = {
+export const textField: Lazy<SingleEditorComponent> = {
   editor: dash.TextFieldEditor,
-  render ({ value }, { update }) {
-    return html`<b-textbox .value="${value.object.value}" .update="${update}"></b-textbox>`
-  },
-  loadDependencies () {
-    return [
-      import('./BulmaTextBox.vue').then(createCustomElement('b-textbox'))
-    ]
+  async lazyRender () {
+    await import('./BulmaTextBox.vue').then(createCustomElement('b-textbox'))
+
+    return ({ value }, { update }) => html`<b-textbox .value="${value.object?.value || ''}" .update="${update}"></b-textbox>`
```

Comment:
I ditched the `loadDependencies` and replaced it with a `lazyRender` which returns the render function. This way it is better to put the whole implementation in a separate module. Should produce more useful bundles